### PR TITLE
runtime comparison of tidymodels and mlr3 gallery post

### PR DESCRIPTION
**Pull request by be-marc**

### CODE REVIEW

File: mlr-org/gallery/technical/2023-10-30-tidymodels/index.qmd [hunk 1]
```diff
@@ -0,0 +1,847 @@
...
+res_2[, framework := factor(framework)]
+res_2 = res_2[, list(time = median(time)), by = c("expr", "framework", "folds")]
+res_2[, time := time / 1e+6]
+
+res_rpart_1_2 = rbindlist(list(
+  sequential = res_1_rpart,
+  parallel = res_2),
+  use.names = TRUE, idcol = "mode")
+
+p1 = ggplot(res_rpart_1_2[c("mlr3_cv3", "tidymodels_cv3", "mlr3_cv6", "tidymodels_cv6", "mlr3_cv9", "tidymodels_cv9"), , on = "expr"],
```
Comment (sebffischer) [inline mlr-org/gallery/technical/2023-10-30-tidymodels/index.qmd:459]: in these plots I would also add plot subtitles (rpart) and (ranger)

File: mlr-org/gallery/technical/2023-10-30-tidymodels/index.qmd [hunk 2]
```diff
@@ -0,0 +1,847 @@
...
+On the other hand, the `tidymodels` package also operates with the same chunk size, but this setting is determined and managed internally within the framework.
+
+By conducting these parallelization tests, we aim to provide a deeper understanding of how each framework handles the distribution and management of computational tasks during the tuning process, particularly in a parallel computing environment.
+This final set of measurements is important in painting a complete picture of the runtime performance of the tune functions across both `tidymodels` and `mlr3` under different operational settings.
+
+```{r}
+options("mlr3.exec_chunk_size" = 200)
+```
+
+Our analysis of the parallelized tuning functions reveals that the runtimes for `mlr3` and `tidymodels` are remarkably similar, indicating a high level of efficiency in both frameworks when operating in a parallelized environment.
```
Comment (sebffischer) [inline mlr-org/gallery/technical/2023-10-30-tidymodels/index.qmd:729]:
```suggestion
Our analysis of the parallelized tuning functions reveals that the runtimes for `mlr3` and `tidymodels` are remarkably similar.
```

Comment (sebffischer) [reply mlr-org/gallery/technical/2023-10-30-tidymodels/index.qmd:729]: non sequitur (maybe both are slow xD)

File: mlr-org/gallery/technical/2023-10-30-tidymodels/index.qmd (hunk 1 above)
Comment (be-marc) [reply mlr-org/gallery/technical/2023-10-30-tidymodels/index.qmd:459]: Unfortunately this is a cowplot to have shared y-axis but i will add a title to each plot

**Pull request merged**